src/proto/tagpost_rpc.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package tagpost;
+
+option java_multiple_files = true;
+option java_package = "com.google.tagpost";
+option java_outer_classname = "TagPostRpcProto";
+
+import "src/proto/tagpost.proto";
+
+service TagpostService {
+  // dummy rpc service
+  rpc FetchMessage(FetchMessageRequest) returns (FetchMessageResponse) {}
+  // fetch all threads by primary tag
+  rpc FetchThreadsByTag(FetchThreadsByTagRequest) returns (FetchThreadsByTagResponse) {}
+  // fetch all comments under one thread
+  rpc FetchCommentsUnderThread(FetchCommentsUnderThreadRequest) returns (FetchCommentsUnderThreadResponse) {}
+}
+
+message FetchMessageRequest {
+  string notice_key = 1;
+  string request_id = 2;
+  string user = 3;
+  string project = 4;
+}
+
+message FetchMessageResponse {
+  string message = 1;
+  repeated string comments = 2;
+}
+
+message FetchThreadsByTagRequest {
+  Tag tag = 1;
+}
+
+message FetchThreadsByTagResponse {
+  repeated Thread threads = 1;
+}
+
+message FetchCommentsUnderThreadRequest {
+  int64 thread_id = 1;
```

Comment:
This should be Thread.

Reply:
Is there any reason we want to use the full Thread proto here? A thread ID seems enough for this feature, and adding more fields in the request message would increase the ambiguity of the interface.

Reply:
I think it is intuitive and scalable if using full thread proto here.

Or FetchThreadsByTagRequest should also use one string instead of the proto.

Reply:
If we want extensibility we can add a `ThreadId` type.

If we use the full `Thread` proto to refer to a thread, we need to specify which fields in `Thread` are used to look up for a thread. Should callers set `primary_tag` in the request? What if the tag they set is different than the actual tag associated with the thread? Then we effectively defined the `ThreadId` type in comments or docs.